entrypoint.sh

```diff
@@ -4,7 +4,7 @@ set -e
 if [ -n "${GITHUB_WORKSPACE}" ] ; then
   cd "${GITHUB_WORKSPACE}" || exit
 fi
-
+git config --global --add safe.directory $GITHUB_WORKSPACE || exit 1
```

Comment:
Thanks, @davefurmidge san

And please change same as [actionlint fix](https://github.com/reviewdog/action-actionlint/pull/66)
(last line are space)

```bash
if [ -n "${GITHUB_WORKSPACE}" ] ; then
  cd "${GITHUB_WORKSPACE}" || exit
  git config --global --add safe.directory "${GITHUB_WORKSPACE}" || exit 1
fi

```

Reply:
I want to keep your contribution, so I want to fix it in this PR.

Reply:
Thank you, changes made as requested